### PR TITLE
disable skipscan by means of put commands

### PR DESCRIPTION
**Pull request by adizaimi**
* can be used in this way in cdb2sql:
  put skipscan enable t1
  put skipscan disable t1
  Note: enable does not work because of code added recently in https://github.com/bloomberg/comdb2/pull/97 (once disabled, it stays disabled).
* Evoke callback for replicants and master to reload analyze stats, to reeval the disable skipscan options.

### CODE REVIEW

**Comment by akshatsikarwar** [comment]
I prefer if this is done using bpfunc and use GET/PUT. This way its SQL and can be run on anynode.

File: db/appsock_handler.c [hunk 1]
```diff
@@ -788,6 +786,7 @@ static void *thd_appsock_int(SBUF2 *sb, int *keepsocket,
                 sbuf2printf(sb, ">disableskipscan cleared\n");
             sbuf2printf(sb, "SUCCESS\n");
             sbuf2flush(sb);
+            bdb_llog_analyze(thedb->bdb_env, 1, &bdberr);
```
Comment (ldennis) [inline db/appsock_handler.c]: is this always on master?

Comment (adizaimi) [inline db/appsock_handler.c]: Good point. It may not--adding check. Thanks.

**Comment by akshatsikarwar** [comment]
Why is there send ? Only get put.

**Comment by adizaimi** [comment]
Akshat, there is no send cmd (I removed it). I added long ago to R6 ability to do: 
   comdb2sc db disableskipscan t1
I could remove it since nothing is using it yet (PR to comdb2sc is pending), and backport put cmd to R6.
I did not add a put cmd -- thought it was useless, just use send("tableparams").

File: tests/analyze.test/t10_01.req.out [hunk 2]
```diff
@@ -1 +0,0 @@
-SUCCESS
```
Comment (ldennis) [inline tests/analyze.test/t10_01.req.out]: Why did you disable my analyze concurrent test??

Comment (adizaimi) [inline tests/analyze.test/t10_01.req.out]: Sorry, bad merge. Will put back asap.

**Pull request merged**